app/views/ppm_estimator/index.html.erb

```diff
@@ -0,0 +1,154 @@
+        </div>
+      </div>
+    </fieldset>
+
+    <fieldset>
+      <legend><div class="ppm-outline"></div>When do you want to move?</legend>
+      <%= form.label :date, 'Moving Date' %>
+      <%= form.date_field :date, { id: :date, pattern: '\d\d\d\d-\d\d?-\d\d?', placeholder: 'yyyy-mm-dd', required: true } %>
+    </fieldset>
+
+    <%= tag.div id: 'entitlements-json', data: { entitlements_json: @entitlements_json } %>
+
+    <fieldset>
+      <legend><div class="ppm-outline"></div>How much, in pounds, do you expect to move without the government's help?</legend>
+      <p>
+        The government will only pay for the actual weight transported, up to your weight allowance.
+        Don't forget - if you move some of your goods yourself (<%= abbr_tag('ppm') %>) and
+        have the government move the rest (<%= abbr_tag('hhg') %>), <em>the weights of both shipments
+        count towards your allowance.</em>
+        <%= link_to 'Need help estimating your total household weight?', weight_estimator_path %>
+      </p>
+      <div id="weight-section">
+        <%= form.label :weight, 'Estimated Household Goods Weight (lbs)' %>
+        <%= form.number_field :weight, { id: :weight, inputmode: 'numeric', required: true } %>
+        <p id="weight-allowance-text">
+          Your weight allowance is up to <strong><span id="entitlement_weight"></span> lbs</strong>.
+        </p>
+      </div>
+      <div id="progear-section">
+        <%= form.label :weight_progear, 'Estimated ProGear Weight (lbs)' %>
+        <%= form.number_field :weight_progear, { id: :weight_progear, inputmode: 'numeric', required: false } %>
+        <p id="progear-allowance-text">
+          Your ProGear allowance is up to <strong><span id="entitlement_progear"></span> lbs</strong>.
+        </p>
+      </div>
+      <div id="progear-spouse-section" hidden>
+        <%= form.label :weight_progear_spouse, "Estimated Spouse's ProGear Weight (lbs)" %>
+        <%= form.number_field :weight_progear_spouse, { id: :weight_progear_spouse, inputmode: 'numeric', required: false } %>
+        <p id="progear-spouse-allowance-text">
+          Your spouse's ProGear allowance is up to <strong><span id="entitlement_progear_spouse"></span> lbs</strong>.
+        </p>
+      </div>
+    </fieldset>
+
+    <fieldset>
+      <legend><div class="ppm-outline"></div>Do you plan to pack your household goods yourself?</legend>
+      <p>
+        In addition to shipping your property yourself, you can also choose to
+        pack your property yourself, and the government will pay you an incentive
+        to do this. The payment can be used to cover things like packing materials
+        and hiring any professional help you may need to pack your house. If
+        there's any money left over, you can keep it!
+        <a href="javascript:$('#alert-packing-incentive').removeAttr('hidden');">How do I make sure the government pays me my packing incentive?</a>
```

Comment:
ditto inline js

Reply:
Samesies comment as previously… This should be a `<button>` and the behavior is better attached in the JS file.